samcli/commands/local/cli_common/invoke_context.py

```diff
@@ -98,6 +106,7 @@ def __init__(self,
         self._debug_port = debug_port
         self._debug_args = debug_args
         self._debugger_path = debugger_path
+        self._parameter_overrides = parameter_overrides or {}
```

Comment:
We set this to be `None` by default everywhere else, should we do that here too?

Reply:
Yes it is set as None everywhere else, but as a good practice I set to empty dictionary if customers don't specify any value.